action.yml

```diff
@@ -53,7 +53,7 @@ runs:
           
           # Filter commit message to just ticket at beginning
           unset TICKET_TO_CHECK
-          TICKET_TO_CHECK=$(git log --max-count=1 --format=%B $commit | tr '[a-z]' '[A-Z]' | sed -nE 's/^([a-zA-Z]{2,}-[0-9]{2,}).*$/\1/p' | head -n 1)
+          TICKET_TO_CHECK=$(git log --max-count=1 --format=%B $commit | tr '[a-z]' '[A-Z]' | sed -nE 's/^([A-Z][A-Z0-9]{1,}-[0-9]{1,}).*$/\1/p' | head -n 1)
```

Comment:
I love this idea, but we lost lower-case ticket numbers. Let's add them back and simplify even further: 
```
TICKET_TO_CHECK=$(git log --max-count=1 --format=%B $commit | tr '[a-z]' '[A-Z]' | sed -nE 's/^([a-zA-Z0-9]{1,}-[0-9]{1,}).*$/\1/p' | head -n 1)
```
Here's the simplified version in action: https://regex101.com/r/ifzsiT/1

How's that look? Please update if it looks good to you and we'll get this merged in.